lib/NormalModule.js

```diff
@@ -648,6 +648,10 @@ class NormalModule extends Module {
 		/** @type {import("../declarations/LoaderContext").NormalModuleLoaderContext<T>} */
 		const loaderContext = {
 			version: 2,
+			/**
+			 * @param {import("../declarations/LoaderContext").Schema} [schema]
```

Comment:
the main purpose of this annotation is to make `schema` parameter optional, so this function can be correctly matched against **both** `getOptions` overloads